torch/export/unflatten.py

```diff
@@ -866,7 +866,14 @@ def remap_input(self, x):
                     self.parent_call_module.insert_arg(0, self.parent.remap_input(x))
             return self.node_to_placeholder[x]
         elif x.op == "call_function" and (
-            x.target == torch.ops.aten.sym_size.int
+            x.target in (
```

Comment:
I think this makes sense with CSE, but just wondering how you ended up with this list?

Reply:
I just collected all the ops required from torchrec :) I made a list because previously this will include every op in the universe (including the embedding lookup custom ops), which will make debugging pretty hard compared to failing fast. 

I agree that in the future we should add a better protection for submodules.

Reply:
I see, I'm a bit surprised with view & diff showing up here? sym_size & item make sense because they're SymInt nodes that get deduped by CSE, and need to be recopied. But those 2 are just tensor compute, which CSE doesn't touch?

Do you know how things are failing there?